Configuration/Applications/python/cmsDriverOptions.py

```diff
@@ -41,21 +41,17 @@ def OptionsFromCommandLine():
 
 def OptionsFromItems(items):
     import sys
-    from Configuration.Applications.Options import parser,threeValued
-    #three valued options
-    for (index,item) in enumerate(items):
-        for (opt,value) in threeValued:
-            if (str(item) in opt) and (index==len(items)-1 or items[index+1].startswith('-')):
-                items.insert(index+1,value)
-                
-    (options,args) = parser.parse_args(items)
-
-    if not options.conditions or options.conditions=="help":
+    from Configuration.Applications.Options import parser
+
+    options = parser.parse_args(items)
+    print(options)
```

Comment:
was this just for debugging?

Reply:
yes, let me just rebase that right out of there...